lib/build.js

```diff
@@ -21,9 +21,24 @@ const TEMPORAL_JWT_TIMEOUT = 12 * 60; // 12 hours in minutes
  * @return {Array}            Array of job ids that match the job names
  */
 function findIdsOfMatchedJobs(jobs, jobNames) {
-    const matchedJobs = jobs.filter(j => jobNames.includes(j.name));
+    return jobs.reduce((match, j) => {
+        let name = j.name;
 
-    return matchedJobs.map(j => j.id);
+        // Get the actual job name if the matching job is a PR job
+        if (j.isPR) {
```

Comment:
I think this needs to be `j.isPR()`
it's a function

Reply:
thx

Reply:
thx